CMakeLists.txt

```diff
@@ -4,14 +4,19 @@ project(NUOPC_DATA_MODELS LANGUAGES Fortran VERSION 0.1)
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 include(ExternalProject)
 find_package(ESMF REQUIRED)
+if (DEFINED PIO)
```

Comment:
This will be still depend on external installation of PIO. Right?

Reply:
Yes, I only added FoX